crates/sovereign-sdk/adapters/mock-da/src/service.rs

```diff
@@ -13,7 +13,7 @@ use sov_rollup_interface::services::da::{BlobWithNotifier, DaService, SlotData};
 use tokio::sync::mpsc::{unbounded_channel, UnboundedSender};
 use tokio::sync::{broadcast, Mutex as AsyncMutex, MutexGuard as AsyncMutexGuard};
 use tokio::time;
-use tracing::{instrument, Instrument};
+use tracing::instrument::Instrument;
```

Comment:
why?

Reply:
idk why but failed to compile all the time.

there is both an instrument attribute and a module